### PR TITLE
Create account feature

### PR DESCRIPTION
**Pull request by cryptokoala4**
Create user works
Registration works 
Navbar login and registration works
Added dropdown menu for district
Logout works

### CODE REVIEW

File: app/views/pages/index.html.erb [hunk 1]
```diff
@@ -11,7 +11,25 @@
       </form>
       <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
-          <input type="text" class="form-control" placeholder="District Name">
+          <select class="form-control">
```
Comment (f3r) [inline app/views/pages/index.html.erb:14]: The best way to having this is to define it in `application_helper.rb` a method that generates this:

``` rb
module ApplicationHelper

def districts_options
  districts = ["Kennedy Town", "HKU", "Sai Ying Pun", "Sheung Wan", "Central", "Admiralty", "Wan Chai", "Causeway Bay", "Tin Hau", "Fortress Hill", "North Point", "Quarry Bay", "Tai Koo", "Sai Wan Ho", "Shau Kei Wan", "Heng Fa Chuen", "Chai Wan"]

  html = '<select class="form-control">'
  districts.each do |district|
    html += "<option>#{district}</option>"
  end
  html += "</select>"

  html
end

end
```

Comment (f3r) [inline app/views/pages/index.html.erb:14]: once you add this method, you can call it in the view with:

``` html
       <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
            <input type="text" class="form-control" placeholder="District Name">
            <%= districts_options %>
            ...
```

**Comment by cryptokoala4** [comment]
Toni will update these fixes on the branch "restaurant" thanks!

**Comment by cryptokoala4** [comment]
image

Hi Fer, we tried implementing the code in different ways but we couldn't get the dropdown menu of the districts to show up. Any thoughts?

**Comment by f3r** [comment]
http://guides.rubyonrails.org/form_helpers.html#making-select-boxes-with-ease :)

**Comment by f3r** [comment]
Also check this: https://github.com/wdi-hk-9/Project-PlaySpace/pull/4/files#diff-c1a79a80fc6f8231ff077d9eb931f477R8

**Pull request merged**